pkg/storage/mysql/mysql.go

```diff
@@ -199,11 +204,11 @@ func (s *Datastore) read(ctx context.Context, store string, tupleKey *openfgav1.
 		sb = sb.Where(sq.Eq{"_user": tupleKey.GetUser()})
 	}
 	if options != nil && options.Pagination.From != "" {
-		token, err := sqlcommon.UnmarshallContToken(options.Pagination.From)
+		token, _, err := s.tokenSerializer.DeserializeContinuationToken(options.Pagination.From)
```

Comment:
should this be a bit more specific or do we see the need for being more generic?

```suggestion
		token, _, err := s.continuationTokenSerializer.Deserialize(options.Pagination.From)
```

Reply:
I like that. If we're not merging any interfaces, this is better.